corehq/apps/enterprise/views.py

```diff
@@ -84,15 +84,21 @@ def platform_overview(request, domain):
 
     context.update({
         'max_date_range_days': EnterpriseFormReport.MAX_DATE_RANGE_DAYS,
-        'reports': [EnterpriseReport.create(slug, request.account.id, request.couch_user) for slug in (
-            EnterpriseReport.DOMAINS,
-            EnterpriseReport.WEB_USERS,
-            EnterpriseReport.MOBILE_USERS,
-            EnterpriseReport.FORM_SUBMISSIONS,
-            EnterpriseReport.ODATA_FEEDS,
-            EnterpriseReport.COMMCARE_VERSION_COMPLIANCE,
-            EnterpriseReport.SMS,
-        )],
+        'groups': [
+            {'name': 'Projects Overview',
```

Comment:
I assume we want to translate the group names here. Otherwise looks good!

Reply:
Thank you! b4a9977077cee152d465a9d496174e9f65fea1be